src/lib/hash/keccak_fips/keccak_fips.h

```diff
@@ -0,0 +1,118 @@
+/*
+* Keccak-FIPS
+* (C) 2010,2016 Jack Lloyd
+* (C) 2023 Falko Strenzke
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_KECCAK_FIPS_H_
+#define BOTAN_KECCAK_FIPS_H_
+
+#include <botan/hash.h>
+#include <botan/secmem.h>
+#include <string>
+
+namespace Botan {
+
+/**
+* KECCAK FIPS
+*/
+
+class Keccak_FIPS_generic : public HashFunction
```

Comment:
Nit: Maybe just `Keccak_FIPS` or `Keccak_FIPS_n`?

Reply:
Good proposal, renaming to `Keccak_FIPS`  comes in next commit.